src/main.py

```diff
@@ -0,0 +1,70 @@
+import json
+import os
+
+from fastapi import FastAPI, Depends, HTTPException
```

Comment:
Hier das Gleiche, auch hier sollten die externen Dependencies in eine `requirements.txt`, sodass diese mit PIP geladen werden können.